pywt/data/_readers.py

```diff
@@ -36,9 +36,9 @@ def ascent():
 
     >>> import matplotlib.pyplot as plt
     >>> plt.gray()
-    >>> plt.imshow(ascent) # doctest: +ELLIPSIS
+    >>> plt.imshow(ascent
```

Comment:
This line looks broken, missing `)`. Shouldn't this fail the doc checking?

Reply:
Ah, interesting. Fixed now.
It actually was not doctested since `pywt.data` is not reachable from the main `pywt` namespace. So the CI now has an explicit `pytest --doctest-modules --pyargs pytest.data`.